src/index.html

```diff
@@ -4,9 +4,460 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+     crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+     rel="stylesheet">
+    <link rel="stylesheet" href="/styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+       class="logo"
+       href="#home">
+        <img
+         src="/images/logo.png"
+         alt="logo"
+         class="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active">
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link">
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link">
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover">
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#" class="nav__link">
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link">
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link">
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="catalog__container">
+        <article class="card" data-qa="card">
+          <img
+            src="/images/imac.jpeg"
+            alt="imac"
+            class="card__image">
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          
+          <p class="card__code">
+            Product code:195434
+          </p>
+
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
```

Comment:
In case this element how no content inside - you can position them closer
```suggestion
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
```